examples/test_vault.ipynb

```diff
@@ -0,0 +1,266 @@
+{
```

Comment:
<p>Does a user <strong>Have to call</strong> <code>create_project_secrets</code>  in order for Vault secrets to be enabled ? </p><p>If so then:</p><ol><li>Fix the wording, it's not <em>can be used </em></li><li>Add it to the code in the cell</li><li>Change the method name to something that is Vault specific - <code>initialize_vault_store</code>  or something like that (this one probably anyways)</li></ol><p>If not, I don't care keeping the method in the SDK in case (for some weird reason) we want to manually trigger initialization, but I don't see a reason talking about it in the example (without stating a good scenario in which it's needed to call it)</p><p>Generally the example should be as simple as possible, in the end you can add some Advanced section with these kind of stuff</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/500/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>It's actually <code>create_vault_secrets</code>  and I have a mistake here. It's used in the cell below that. It does initialization and adds the secrets for that project. And, yes, it needs to be called if it wasn't called for the specific project before that (unless you manually created everything you need, which is highly unlikely). So, I'll fix and rephrase.</p>